share/spice/economic_indicators/economic_indicators.js

```diff
@@ -0,0 +1,93 @@
+(function (env) {
+    "use strict";
+    function getDataFromAPIResponse(api_response){
+        //validate if api response is null
+        if(api_response === null || api_response === undefined){
+            return null;
```

Comment:
We need to return `Spice.failed` like in https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/alternative_to/alternative_to.js#L7